frontend/src/pages/accounts/profile.module.scss

```diff
@@ -48,31 +48,112 @@
     }
 
     .account-stats {
-      display: flex;
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+      gap: $spacing-md;
       text-align: center;
       justify-content: space-evenly;
 
+      @media screen and #{$mq-md} {
+        display: flex;
+      }
+
       .stat {
+        flex: 1 0 auto;
         display: flex;
         flex-direction: column;
         align-items: center;
         justify-content: space-between;
-        padding: 0 $spacing-md;
-        border-right: 1px solid $color-light-gray-20;
-
-        &:last-child {
-          border-right: 0 none transparent;
+        gap: $spacing-xs;
+        padding: $spacing-md;
+        background-color: $color-white;
+        border: 1px solid $color-light-gray-20;
+        border-radius: $border-radius-md;
+
+        @media screen and #{$mq-lg} {
+          // On small screens, stats have a border and all that, but since
+          // Protocol doesn't have max-width media queries, we need to unset
+          // all those for larger screens:
+          border-style: none;
+          border-right-style: solid;
+          border-radius: 0;
+          background-color: transparent;
+          padding-block: 0;
+
+          &:last-child {
+            border-right-style: none;
+          }
         }
 
         .label {
+          @include text-body-sm;
           color: $color-dark-gray-10;
         }
 
         .value {
-          @include text-title-sm;
+          @include text-title-xs;
           font-family: $font-stack-firefox;
           font-weight: 700;
           padding: $spacing-xs 0;
+          flex: 1 0 auto;
+          display: flex;
+          flex-direction: column;
+
+          .learn-more-wrapper {
+            @include text-body-sm;
+            position: relative;
+            font-weight: 400;
+
+            .open-button {
+              background-color: transparent;
+              border-style: none;
+              border-radius: $border-radius-lg;
+              color: $color-blue-50;
+              padding-inline: $spacing-md;
+              font-family: $font-stack-base;
+              font-weight: 500;
+              cursor: pointer;
+            }
+
+            &.is-open .open-button {
+              background-color: $color-grey-10;
+            }
+
+            .close-button {
+              align-self: end;
+              background-color: transparent;
+              border-style: none;
+              border-radius: $border-radius-sm;
+              float: right;
+              padding: 0;
+              cursor: pointer;
+
+              &:hover {
+                color: $color-blue-50;
+              }
+            }
+
+            .learn-more-tooltip {
+              position: absolute;
+              top: $spacing-xl;
+              right: 0;
+              padding: $spacing-md;
+              background-color: $color-white;
+              width: $content-sm;
+              max-width: 80vw;
```

Comment:
OK, so I remembered the problem, and I'm not sure if there's a good fix. I at least added a comment explaining the magic number (85a4043e), but if you have an idea for a more robust solution, I'd love to try it.

Reply:
Actually, thanks for calling me out on this, because I did figure it out now, and caught an eventual bug in the process. The changes I made:

- I realised that instead of positioning it relative to the wrapper around the "Learn more" button, I could instead position it relative to the `.stat` element containing it - that solved the problem of not knowing the distance from the side of the screen.
- I then also realised that the element might be on the left-hand side of the screen when we add/remove other stats (e.g. replies). Thus I also made it left-aligned when needed.
- And then I realised that react-aria also has [`useOverlayPosition`](https://react-spectrum.adobe.com/react-aria/useOverlayPosition.html) to make sure it's properly anchored to the button, because the top margin wasn't quite accurate there either.

89ff5858 and 7ad06405

I also added a hover state as per Eduardo's request: 05dc17bc